io/disk/multipath_test.py

```diff
@@ -40,6 +41,10 @@ def setUp(self):
         """
         Set up.
         """
+        self.policy = self.params.get('policy', default='service-time')
+        self.policies = ["service-time", "round-robin", "queue-length"]
+        self.policies.remove(self.policy)
+        self.policies.append(self.policy)
```

Comment:
why removed the policy and added it Back?

Reply:
@Naresh-ibm to make it as the last member of the list.
So that in the actual policy change test, we change all to policies and then come back to current policy.

Reply:
Added this as a comment too.